fdbserver/DDTxnProcessor.actor.cpp

```diff
@@ -524,7 +517,7 @@ class DDTxnProcessorImpl {
 					TraceEvent(SevDebug, "WaitForDDEnabled")
 					    .detail("Mode", m)
 					    .detail("IsDDEnabled", ddEnabledState->isEnabled());
-					if (m && ddEnabledState->isEnabled()) {
+					if (m != 0 && ddEnabledState->isEnabled()) {
```

Comment:
Why do we need to change from `m` to `m != 0`?

Reply:
No. Revised.